spec/services/users/invitation_service_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Users::InvitationService do
+  let!(:current_user) { create(:user) }
+  let(:invite_params) do
+    {
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      phone: phone,
+      email: Faker::Internet.email,
+      member_type: User.member_types.keys.sample,
+      avatar: Rack::Test::UploadedFile.new(ENV['LOCAL_IMAGE_PATH']),
+      address: Faker::Address.full_address
+    }
+  end
+
+  before do
+    allow(Twilio::TwilioTextMessenger).to receive(:perform)
+  end
+  subject { Users::InvitationService.perform(
+      invite_params.merge(current_user: current_user)) }
```

Comment:
use `described_class` instead of `Users::InvitationService`

Reply:
fixed it and founded same mistakes in family_presenter_spec.rb